lib/challenge.js

```diff
@@ -177,14 +206,16 @@ function prompt9() {
  * Write a loop that pushes every even number between 0 and 100 into an array,
  * then print the array.
  */
-function prompt10() {
-  const myArray = [];
-  // YOUR CODE HERE
+//function prompt10() {
+ // const myArray = [];
+ for (let i = 0; i <= 100; i ++) {
+  console.log(i);
+}
```

Comment:
Challenge yourself to revisit this.  We are trying to push every even number into an array and then print that array to the console.  Think about prompt 7 (resubmission not required)

Reply:
I added 11, i will revisit 7. I broke something big in my code for vscode that i've yet to figure out. I'm unable to debug and my console returns errors.